src/Library/demos/Dialogs/main.js

```diff
@@ -0,0 +1,69 @@
+import Adw from "gi://Adw";
+import Gtk from "gi://Gtk";
+
+const button_simple = workbench.builder.get_object("button_simple");
+const button_advanced = workbench.builder.get_object("button_advanced");
+const window = button_simple.get_ancestor(Gtk.Window);
+
+function createSimpleDialog() {
+  let dialog = new Adw.MessageDialog({
+    heading: "Save Changes?",
+    body: "Opened files have unsaved changes. Unsaved changes will be lost forever!",
+    close_response: "cancel",
+    modal: true,
+  });
```

Comment:
this is not really a pattern we would use

can you reuse the heading/body of the HIG example https://developer.gnome.org/hig/patterns/feedback/dialogs.html?highlight=dialog#message-dialogs ?

Reply:
Yes I can. I want to follow HIG and will be adding the error dialog as well. I was wondering if you could clear some things up for me. 

Why wouldn't we use this pattern? I mimicked the message dialog example from the Adwaita Demo. I liked the example as it shows how message dialogs have up to 3 responses and shows off each of the response appearance types.

This example on the hig patterns only shows 2 of the appearance types.
![image](https://user-images.githubusercontent.com/103920890/223772446-5a171ac6-9c62-4570-bd85-f89a7b661de7.png)

Reply:
Nevermind, I realized I used suggested response for the login example